pkg/cloud/services/network/vpc_test.go

```diff
@@ -104,40 +105,39 @@ func TestReconcileVPC(t *testing.T) {
 							Values: aws.StringSlice([]string{ec2.VpcStatePending, ec2.VpcStateAvailable}),
 						},
 					},
-				})).
-					Return(&ec2.DescribeVpcsOutput{
-						Vpcs: []*ec2.Vpc{
-							{
-								State:     aws.String("available"),
-								VpcId:     aws.String("vpc-exists"),
-								CidrBlock: aws.String("10.0.0.0/8"),
-								Tags: []*ec2.Tag{
-									{
-										Key:   aws.String("sigs.k8s.io/cluster-api-provider-aws/role"),
-										Value: aws.String("common"),
-									},
-									{
-										Key:   aws.String("Name"),
-										Value: aws.String("test-cluster-vpc"),
-									},
-									{
-										Key:   aws.String("sigs.k8s.io/cluster-api-provider-aws/cluster/test-cluster"),
-										Value: aws.String("owned"),
-									},
+				})).Return(&ec2.DescribeVpcsOutput{
+					Vpcs: []*ec2.Vpc{
+						{
+							State:     aws.String("available"),
+							VpcId:     aws.String("vpc-exists"),
+							CidrBlock: aws.String("10.0.0.0/8"),
+							Tags: []*ec2.Tag{
```

Comment:
Can we extract this Tag slice to a variable and declare globally so that it can be used in other func as well?

Reply:
@Madhur97 Is this not taken care?

Reply:
Have not raised the revision yet. All comments will be resolved in new revision.